content/applications/finance/accounting/customer_invoices/credit_notes.rst

```diff
@@ -2,102 +2,111 @@
-   * the goods delivered are damaged
+ - a mistake in the invoice;
+ - a return of the goods, or a rejection of the services;
+ - the goods delivered are damaged.
 
 .. note::
-   Issuing a credit note is the only legal way to cancel,
-   refund or modify a validated invoice. Don’t forget to
-   *register the payment* afterward if you need to send money
-   back to your customer.
+   Issuing a credit/debit note is the only legal way to cancel, refund, or modify a validated
+   invoice. Do not forget to **register the payment** afterward if you need to send money back to
+   your customer.
 
-Issue a Credit Note
+Issue a credit note
 ===================
 
-You can create a credit note from scratch by going to
-:menuselection:`Accounting --> Customers --> Credit Notes`,
-and by clicking on *Create*. Filling the Credit Note’s form
-works the same way as the Invoice’s form.
+You can create a credit note from scratch by going to :menuselection:`Accounting --> Customers -->
+Credit Notes`, and by clicking on :guilabel:`Create`. Filling a credit note form works the same way
+as an invoice form.
 
-However, most of the time, credit notes are generated directly
-from the invoices they are related to.
-
-To do so, open the *Customer Invoice*, and click on *Add Credit Note*.
+However, most of the time, credit notes are generated directly from the invoices they are related
+to. To do so, go to :menuselection:`Accounting --> Customers --> Invoices`, open the related
+**customer invoice**, and click on :guilabel:`Add Credit Note`.
 
 .. image:: credit_notes/credit_notes01.png
-   :align: center
+   :alt: Add a credit note to a customer invoice.
 
 You can choose between three options:
-   - Partial Refund
-   - Full Refund
-   - Full refund and new draft invoice
+   - :guilabel:`Partial Refund`;
+   - :guilabel:`Full Refund`;
+   - :guilabel:`Full refund and new draft invoice`.
 
 .. note::
-   Credit Notes’ numbers start with “R” and are followed by the
-   number of the document they are related to (e.g., RINV/2019/0004).
+   Credit notes referencing start with “R” and are followed by the number of the document they are
+   related to (e.g., RINV/2019/0004). In this case, the invoice INV/2019/0004.
 
-Partial Refund
+Partial refund
 --------------
 
-Odoo creates a draft credit note already prefilled with all the
-necessary information from the original invoice.
-
-This is the option to choose to do a partial refund, or if you
-want to modify any detail on the credit note.
+When selecting the :guilabel:`Partial Refund` option, Odoo creates a draft credit note already
+prefilled with all the necessary information from the original invoice. This is the option to choose
+if you wish to do a partial refund, or if you want to modify any detail of the credit note
 
 .. note::
-   This is the only option available for invoices that are already marked as *Paid*.
+   This is the only option available for invoices that are already marked as *paid*.
```

Comment:
```suggestion
   This is the only option available for invoices that are already marked as *in payment* or *paid*.
```
Or we could say "invoices that are fully paid" - but I think it's important to not just say "marked as paid" because that means they've been reconciled too which isn't necessary.